electrum/gui/qt/create_new_stake_window.py

```diff
@@ -0,0 +1,472 @@
+        self.label_6.setText(_("Predicted Rewords:"))
+        self.label_6.setMaximumSize(QtCore.QSize(16777215, 30))
+        self.label_6.setBaseSize(QtCore.QSize(0, 30))
+        self.vl_rewords.addWidget(self.label_6)
+        self.ep_label = QtWidgets.QLabel(self.verticalLayoutWidget)
+        self.ep_label.setText(
+            _("Estimated payout: ") +
+            str(self.spinBox_amount.value() * self.period['days'] * 0.21) +
+            ' ELCASH')
+        self.ep_label.setMaximumSize(QtCore.QSize(16777215, 20))
+        self.vl_rewords.addWidget(self.ep_label)
+        self.Main_v_layout.addLayout(self.vl_rewords)
+        self.description2_label = QtWidgets.QLabel(self.verticalLayoutWidget)
+        self.description2_label.setText(_("Click Next to go confirmation view. "))
+        self.description2_label.setMaximumSize(QtCore.QSize(16777215, 50))
+        self.Main_v_layout.addWidget(self.description2_label)
+        self.gridLayout_2 = QtWidgets.QGridLayout()
+        self.gridLayout_2.setSpacing(2)
+        self.terms_button = QtWidgets.QPushButton(self.verticalLayoutWidget)
+        self.terms_button.setText(_("Terms & Conditions"))
+        self.terms_button.setMaximumSize(QtCore.QSize(140, 16777215))
+        font = QtGui.QFont()
+        font.setUnderline(True)
+        self.terms_button.setFont(font)
+        self.terms_button.setText(_("Terms & Conditions"))
+        self.terms_button.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
+        self.terms_button.setStyleSheet("border: none;")
+        self.terms_button.setAutoDefault(True)
+        self.gridLayout_2.addWidget(self.terms_button, 0, 1, 1, 1)
+        self.horizontalLayout = QtWidgets.QHBoxLayout()
+        spacer_item1 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
+        self.horizontalLayout.addItem(spacer_item1)
+        self.cancel_button = QtWidgets.QPushButton(self.verticalLayoutWidget)
+        self.cancel_button.setText(_("Cancel"))
+        self.cancel_button.clicked.connect(self.on_push_cancel_button)
+        self.cancel_button.setMaximumSize(QtCore.QSize(60, 16777215))
+        self.cancel_button.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
+        self.horizontalLayout.addWidget(self.cancel_button)
+        self.next_button = QtWidgets.QPushButton(self.verticalLayoutWidget)
+        self.next_button.setText(_("Next"))
+
+        if not self.valid_enough_coins(self.MIN_AMOUNT):
+            self.next_button.setEnabled(False)
+
+        self.next_button.setMaximumSize(QtCore.QSize(60, 16777215))
+        self.next_button.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
+        self.next_button.clicked.connect(self.on_push_next_button)
+        self.horizontalLayout.addWidget(self.next_button)
+        self.gridLayout_2.addLayout(self.horizontalLayout, 0, 5, 1, 2)
+        self.Main_v_layout.addLayout(self.gridLayout_2)
```

Comment:
We need to split this chunk into smaller bits. If these section is only for construction we can use Inner Functions.